core/primitives/src/trie_key.rs

```diff
@@ -50,8 +48,15 @@ pub mod col {
     /// This column id is used when storing the postponed PromiseYield receipts
     /// (`primitives::receipt::Receipt`).
     pub const PROMISE_YIELD_RECEIPT: u8 = 12;
-    /// All columns except those used for the delayed receipts queue and the yielded promises
-    /// queue, which are both global state for the shard.
+    /// Indices of outgoing receipts. A singleton per shard.
+    /// (`primitives::receipt::BufferedReceiptIndices)
```

Comment:
nit: Missing backtick at the end. Also I think in rust the sytanx is [`Thing`] for IDE to pick up on it but I'm fine with it for consistency with the rest.